### ISSUE TITLE
[macOS] Closing square bracket display incomplete in "Project Explorer" view

### ISSUE THREAD

**Issue by quaff** [eclipse-platform/eclipse.platform.ui#1430]
The project is collapsed:
![1](https://github.com/eclipse-platform/eclipse.platform.ui/assets/143040/2f609f2a-9d9a-481e-8db7-1a8626626f38)

The project is expanded:
![2](https://github.com/eclipse-platform/eclipse.platform.ui/assets/143040/51f77af8-7642-438f-90cc-2110aafb3c78)

## Tested under this environment:
* OS: macOS 14.2 (23C64)
* Eclipse IDE/Platform version (as shown in *Help > About*): Version: 2023-12 (4.30.0)

**Comment by tomaswolf** [comment]
Compare PR eclipse-platform/eclipse.platform.swt#969. Same issue?

**Comment by Phillipus** [comment]
> Compare PR [eclipse-platform/eclipse.platform.swt#969](https://github.com/eclipse-platform/eclipse.platform.swt/pull/969). Same issue?

Yes, this will be it. The Eclipse 4.30 mac launcher is built using a later SDK and this causes this problem.

**Comment by BeckerWdf** [comment]
Can we close this as fixed as https://github.com/eclipse-platform/eclipse.platform.swt/pull/969 is closed?
